docs/wiki.md

```diff
@@ -1,59 +1,66 @@
+### NuGet package
 
-<img src = "https://github.com/dotnet/roslyn-analyzers/blob/main/docs/SuppressErrorList.PNG">
+To get the analyzers from the NuGet package, you need to add the package in your `csproj` or `Directory.Build.props` in an `ItemGroup` as follows:
 
-## Severity Levels for Analyzers
-* **None** – rule violations at this severity level will not have a visual adornment in the editor nor appear in the Error List/scroll bar.
-* **Suggestion** – rule violations at this severity level will be adorned with a gray, dotted line and will appear as "Messages" in the Error List and gray boxes in the scroll bar.
-* **Warning** – rule violations at this severity level will be adorned with a green, squiggly line and will appear as "Warnings" in the Error List and green boxes in the scroll bar. 
-* **Error** – rule violations at this severity level will be adorned with a red, squiggly line and will appear as "Errors" in the Error List and red boxes in the scroll bar. 
- 
-## FAQ
-* **Why am I only seeing rule violations in open files rather than my entire solution?**
+```xml
+<PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="PackageVersionGoesHere" PrivateAssets="all" />
+```
 
-This is because "Full Solution Analysis" is off by default. To turn on live code analysis for your entire solution, go to Tools>Options>Text Editor> [C# | Basic] > Advanced > Enable full solution analysis.
+### .NET SDK
 
-* **Do I need a .ruleset file in every project?**
+Starting in .NET 5, these analyzers are included with the .NET SDK. If your project targets .NET 5 or later, code analysis is enabled by default. If your project targets a different .NET implementation, for example, .NET Core, .NET Standard, or .NET Framework, you must manually enable code analysis by setting the [EnableNETAnalyzers](https://learn.microsoft.com/dotnet/core/project-sdk/msbuild-props#enablenetanalyzers) property to `true`.
 
-Yes. Ruleset files work on a per-project basis. When we move to EditorConfig, the configuration file will apply at the directory-level.
+### Visual Studio extension (VSIX) (deprecated)
 
-* **How do I report issues/bugs I find? How do I report a false-positive?**
+You can also get the analyzers as a Visual Studio extension ([VS2017](https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.MicrosoftCodeAnalysis2017) or [VS2019](https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.MicrosoftCodeAnalysis2019)). This is deprecated in VS 2022.
+
+## Apply a code fix for a rule
+
+Place your cursor in the span of the squiggle/dotted line and press `Ctrl+.` to trigger the lightbulb menu. From the lightbulb, you will see all available quick actions.
```

Comment:
Please avoid all the text here and instead just add a help link to https://learn.microsoft.com/dotnet/fundamentals/code-analysis/suppress-warnings